### PR TITLE
Qc curves endpoint

### PR DESCRIPTION
**Pull request by local-minimum**
And endpoint to request all growth-curve related data for a plate. Request takes roughly 2s.

Updated fixture to reuse fixture files across tests.

### CODE REVIEW

File: scanomatic/ui_server/qc_api.py [hunk 1]
```diff
@@ -11,10 +11,11 @@
 import uuid
 
 from dateutil import tz
-from flask import Flask, jsonify, request, send_from_directory
+from flask import Flask, jsonify, request, send_from_directory, Blueprint
```
[the remark this answers is not in the document]
Comment (local-minimum) [reply scanomatic/ui_server/qc_api.py]: fixed

File: scanomatic/data_processing/analysis_loader.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+class AnalysisLoader(object):
+    def __init__(self, project):
+        self.project = project
+        self.path = convert_url_to_path(project)
+
+    @property
+    def times(self):
+        try:
+            return np.load(os.path.join(self.path, 'phenotype_times.npy'))
```
Comment (skymandr) [inline scanomatic/data_processing/analysis_loader.py]: At some point we should probably try to move away from saving in npy.

File: tests/unit/data_processing/test_analysis_loader.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    def test_raises_exception_if_smoth_are_missing(self):
+        loader = AnalysisLoader('test')
+        with pytest.raises(CorruptAnalysisError):
+            loader.smooth_growth_data
+
+    def test_returns_reqested_plate(self, mock_convert_url_to_path):
+        loader = AnalysisLoader('test')
+        plate = loader.get_plate_data(1)
+        expect = [
+            28670.644118, 30694.83756341, 31778.23297606, 31825.43530291,
```
Comment (skymandr) [inline tests/unit/data_processing/test_analysis_loader.py]: Unecessary to have different line break conventions for the two expects, please fix.

File: scanomatic/data_processing/analysis_loader.py (hunk 2 above)
Comment (local-minimum) [reply scanomatic/data_processing/analysis_loader.py]: This is why this code is in its own module. :nerd_face:

**Comment by local-minimum** [comment]
Frontend is part of integration-task I'm working on.

**Pull request merged**